src/shogun/base/SGObject.cpp

```diff
@@ -547,6 +547,17 @@ void CSGObject::init()
 	watch_method("num_subscriptions", &CSGObject::get_num_subscriptions);
 }
 
+std::string CSGObject::get_description(const std::string &name) const
+{
+	auto it = this->get_params().find(name);
```

Comment:
why `this->`?

Reply:
I usually use it to show that `get_params` is a member function and not a method declared elsewhere :) it is more a personal choice rather than a standard (shouldn't be impacting the performance or?).

Reply:
OK! No it doesn't affect performance, I was just wondering :)